rcl/include/rcl/client.h

```diff
@@ -256,7 +256,8 @@ rcl_send_request(const rcl_client_t * client, const void * ros_request, int64_t
  * cannot be checked by this function and therefore no deliberate error will
  * occur.
  * The request_header is an rmw struct for meta-information about the request
- * sent (e.g. the sequence number).
+ * sent (e.g. the sequence number). The caller must provide a pointer to an
```

Comment:
nitpick: each new sentence on it's own line.

Reply:
Fixed fea711ce38f62729367b6cf9de7bdafa6d10941f